swift3/acl_handlers.py

```diff
@@ -388,9 +387,20 @@ def HEAD(self, app):
     def PUT(self, app):
         container = self.req.container_name + MULTIUPLOAD_SUFFIX
         obj = '%s/%s' % (self.obj, self.req.params['uploadId'])
-        resp = self.req._get_response(app, 'HEAD', container, obj)
-        self.req.headers[sysmeta_header('object', 'acl')] = \
-            resp.sysmeta_headers.get(sysmeta_header('object', 'tmpacl'))
+
+        if self.req.environ.get('swift3.acl-cache'):
```

Comment:
I'm not sure if this is supposed to hold a cache or a cache entry...
If it holds a cache, it should be named `swift3.acl_cache` (with an underscore, like all other keys).
If it holds a cache entry, it should be named `swift3.cached_acl`.

Reply:
The `swift3.cached_acl` is the remains of a previous version. The test can be removed